src/primitives.jl

```diff
@@ -12,14 +12,67 @@ $CALC_EXTENT_KEYWORD
 
 # This file mainly defines the [`apply`](@ref) function.
 
+#=
+## What is `apply`?
+
+`apply` applies some function to every geometry matching the `Target`
+GeoInterface trait, in some arbitrarily nested object made up of:
+- `AbstractArray`s 
+- Some arbitrary iterables may also work here
```

Comment:
I am not really sure what this is referring too

Reply:
You mean the dot points? They are the types that will be accepted and recursed over by `apply`.

Maybe I can make that clearer.

Reply:
Or you mean the iterables? it doesn't quite sit with the others but anything unknown (not trait and not an AbstractArray) we try to iterate over.

Reply:
Okay yeah that makes sense.

Reply:
Ok I'll try to work that into the text